packages/script/src/errors.ts

```diff
@@ -28,21 +24,7 @@ export class ScriptResultDecoderError extends Error {
           )
           .join('\n')}`
       : null;
-    const logReceipts = result.receipts.filter(
-      (r) => r.type === ReceiptType.Log || r.type === ReceiptType.LogData
-    ) as Array<TransactionResultLogReceipt | TransactionResultLogDataReceipt>;
-    const logsText = logReceipts.length
-      ? `Logs:\n${logReceipts
-          .map(({ type, id, ...r }) =>
-            printLineWithId(
-              id,
-              `${
-                type === ReceiptType.LogData ? (r as TransactionResultLogDataReceipt).data : r.val0
-              }`
-            )
-          )
-          .join('\n')}`
-      : null;
+    const logsText = logs.length ? `Logs:\n${JSON.stringify(logs, null, 2)}` : null;
```

Comment:
Is there any other place/test validating this change?

I presume [this test](https://github.com/FuelLabs/fuels-ts/pull/640/files#diff-39d1aa6104e1987397125435435ff592c6b7057ff4b64ea578709fa37610716d
) `log` deconstructed variable, but I wonder if we didn't have anything in place for this case before that, as nothing seems to have been broken.